### ISSUE TITLE
Incorrect display of Sum value of regionalized values in Calculations Tab

### ISSUE THREAD

**Issue by tomeli5n** [dbeaver/dbeaver#22626]
### Description

In the course of executing the following SQL query:

```sql
select FORMAT(sum(42998), 0, "es_AR") as number
UNION ALL
SELECT FORMAT(sum(475093), 0, "es_AR") as number
UNION ALL
SELECT FORMAT(sum(166999), 0, "es_AR") as number
```

This result is obtained:

![image](https://github.com/dbeaver/dbeaver/assets/7417323/d4d22153-286c-4190-8a55-8772df88a380)

The obtained result, as illustrated in the provided image, shows the sum as 685,09 instead of the anticipated 685090.

![image](https://github.com/dbeaver/dbeaver/assets/7417323/6eead3ea-1c69-4a59-b41b-7ebce0d8a329)

### DBeaver Version

Community Versión 23.3.2.202401072024

### Operating System

W10

### Database and driver

MariaDB

### Steps to reproduce

Run the query and select the values

**Comment by LonwoLonwo** [comment]
Hello @tomeli5n 

Unfortunately, I need more context to understand why this result is incorrect for your opinion.

![2024-01-29 19_15_40-Desmos _ Научный калькулятор — Mozilla Firefox](https://github.com/dbeaver/dbeaver/assets/45152336/cd71768d-2993-445e-b746-87c1578324db)

**Comment by LonwoLonwo** [comment]
Hello @tomeli5n 

Please provide a screen from your Preferences -> Editors -> Data Editor -> Data Formats page.

**Comment by tomeli5n** [comment]
![image](https://github.com/dbeaver/dbeaver/assets/7417323/4f622127-9d8d-4874-80c6-311223787d66)

I just noticed that I am performing additions on text values, results of the FORMAT function. Therefore, it would be the correct calculation

**Comment by LonwoLonwo** [comment]
Okey, thanks. We should check the data formatting profile before the string -> number conversation. Maybe it can be a customizable choice.
